CP2077 - EasyInstall/Form1.cs

```diff
@@ -366,5 +378,63 @@ private void btnUninstall_Click(object sender, EventArgs e)
                 MetroFramework.MetroMessageBox.Show(this, "Uninstall was not able to delete the mod. Just delete <cyberpunk install path>/bin/x64/version.dll and <cyberpunk install path>/bin/x64/plugins/", "Exception!", MessageBoxButtons.OK, MessageBoxIcon.Error);
             }
         }
+        private void btnFindSteam_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                string path = SteamGamePath.FindGameByAppID("1091500");
+                if (path == null)
+                {
+                    MetroFramework.MetroMessageBox.Show(this, "Error: Couldn't Find CyberPunk for Steam!", "File not found Error!", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    //MessageBox.Show("Error: Couldn't Find CyberPunk for steam!");
+                    return;
+                }
+                DialogResult result = MessageBox.Show(path, "Is this Correct?", MessageBoxButtons.YesNo);
+                if (result == DialogResult.Yes)
+                    PatchGame(path);
+                else if (result == DialogResult.No)
+                    MetroFramework.MetroMessageBox.Show(this, null, "Install Canceled.", MessageBoxButtons.OK);
+                else
+                {
+                    MetroFramework.MetroMessageBox.Show(this, "Error during installation\nError code: 2", "Critical Error!", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    btnMain.Text = "Critical Error!";
+                }
+            }
+            catch (Exception ex)
+            {
+                MetroFramework.MetroMessageBox.Show(this, "Error: " + ex, "Unknown Error!", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                //MessageBox.Show("Error" + ex);
+            }
+        }
+
+        private void btnFindGoG_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                string path = GoGGamePath.FindGameByAppID("1423049311");
```

Comment:
Hi all,

I've tested out the auto find with the GoG version of the game and while it returns the root Cyberpunk installation, it's not the correct path for the files to be placed in.

We either need to return the path with `\bin\x64` already appended, or append it to the `PatchGame(path)` method below.